src/qrdrop.js

```diff
@@ -137,7 +137,7 @@ function validateArgs () {
 }
 
 function isNotIntegerAboveZero (value) {
-    return !(Number.isInteger(value) && value > 0);
+    return !(value > 0);
```

Comment:
```suggestion
    return !(Number.isInteger(value) && value > 0);
```

seems like Number.isInteger check is significant here